csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -1251,7 +1251,7 @@ Audience of note (`audience`) of value type `string` with 1 or more characters i
     safety engineers
 ```
 
-Note category (`category`) of value type `string` and `enum` indicates the choice of what kind of note this is.
+Note category (`category`) of value type `string` and `enum` contains the information of what kind of note this is.
```

Comment:
Nit: or, simply ˋnames the category the note belongs tò?

Reply:
I tried to avoid the "word is explained by itself"...

Reply:
Sometimes words are chosen for a reason :wink: - "containing information of  what kind" to me blurs what the well-chosen term `category` achieved, but I am OK either way